puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/systest.shell_template

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -xe
```

Comment:
What makes this no longer runnable as a stand-alone shell?

Reply:
It is no longer runnable as a standalone shell script because all shell variables have to be escaped with an extra set of `{}` to avoid being mistaken for builder macros.

if bash doesn't have a problem with `${{os}}` vs `${os}` then I simply made an assumption that was incorrect.

Reply:
It is weird that `!include-raw` still applies a template over it. The docs suggest `!include-raw-escape` would escape it for you while still being a plain shell script in the repo. Could you try that?